### PR TITLE
RM-7446: Right textbox border cutoff since Chrome v83

### PR DESCRIPTION
**Pull request by iam-mholle**
https://re-motion.atlassian.net/browse/RM-7446

Reference screenshots are not yet updated.

### CODE REVIEW

**Comment by MichaelKetting** [comment]
Regarding the question if we should use width:calc-2px or add a padding, I'll need to see this live. Based on the existing css, we already use calc for calculating widths/heights, so it's jsut a question if padding or width is the correct / less-intrustive option here.

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css [hunk 1]
```diff
@@ -24,7 +24,8 @@ span.bocAutoCompleteReferenceValue input
 /* Sets width for Chromium browsers */
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css:24]: .blink already says that this is for chromium. If we want a comment, we should document what the workaround fixes.

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css [hunk 2]
```diff
@@ -202,3 +182,11 @@ span.bocTimeInputWrapper input
   height: 1.2em;
   line-height: 1em;
 }
+
+/* Sets width for Chromium browsers */
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocDateTimeValue.css:186]: see above.

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css [hunk 3]
```diff
@@ -24,7 +24,8 @@ span.bocAutoCompleteReferenceValue input
 /* Sets width for Chromium browsers */
 .blink span.bocAutoCompleteReferenceValue input
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocAutoCompleteReferenceValue.css:25]: should we move this to the bottom of the file, similiar to the other controls?

File: Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocTextValue.css [hunk 4]
```diff
@@ -70,7 +70,8 @@ span.bocTextValue input
 /* Sets width for Chromium browsers */
```
Comment (MichaelKetting) [inline Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocTextValue.css:70]: see above

**Comment by MichaelKetting** [comment]
please also add a generic input[type=text] and type=password to style.css. I know it's redundant, but this will catch all the other textboxes that might exist.

**Comment by iam-mholle** [comment]
Added to master in d67113c3abdeabb191152fcad044789d5c4cad77.

**Pull request closed without merging**